### PR TITLE
Fixed #33481 -- Clarified remove_stale_contenttypes data loss warning.

### PR DESCRIPTION
**Pull request by Waheedsys**
updated remove_stale_contenttypes data loss warning

### CODE REVIEW

**Comment by shangxiao** [comment]
The requested change is changing the meaning of the message to something less correct 🤷

**Comment by Waheedsys** [comment]
so what should i do to improve it

**Comment by shangxiao** [comment]
Ah sorry this is for Ticket 33481, reopening.

I'll add a review.

**Review by shangxiao — CHANGES_REQUESTED** [review]
There's a couple of important things to do when contributing to Django:
 - First please make sure that all lint issues are addressed
 - Also please don't comment out lines, just replace the old message with the new message.

Having said that the message is slightly more incorrect because as [Tim pointed out in the ticket](https://code.djangoproject.com/ticket/33481#comment:3) "Triggers and constraints" is not really what the message is talking about – it's talking about cascade deletes.

Please reword it to something more appropriate 👍

**Comment by Waheedsys** [comment]
thank you i will

**Comment by Waheedsys** [comment]
is "The list excludes deletions caused by manually added database constraints or triggers." better

**Comment by timgraham** [comment]
I think the ticket reporter's confusion was interpreting "Django's models" as meaning models in "django.contrib". Perhaps it would be more clear to say "Django models".

**Comment by timgraham** [comment]
You have unnecessary edits. Please simply remove " 's " after "Django" and leave the other lines unchanged.

**Comment by felixxm** [comment]
@Waheedsys Thanks :+1: Welcome aboard :boat:
